src/bbs.c

```diff
@@ -968,9 +938,9 @@ bbs_proof_prf (
 {
 	// All these have length 17
 	static uint8_t *dsts[] = {
-		(uint8_t*) "random msg scalar", (uint8_t*) "random r_1 scalar",
-		(uint8_t*) "random r_2 scalar", (uint8_t*) "random e_t scalar",
-		(uint8_t*) "random r1t scalar", (uint8_t*) "random r3t scalar",
+		(uint8_t*) "random msg scalar", (uint8_t*) "random r_1 scalar", (uint8_t*)
```

Comment:
Uncrustify formates arguments weirdly, should i manually change lines like this or adapt the .uncrustify.cfg? @bellebaum